### ISSUE TITLE
Scrollbar on right side is not clickable "at the edge", instead yields resize grabber

### ISSUE THREAD

**Issue by hlascelles** [standardnotes/desktop#755]
Using v3.8.21 (and possibly older versions), I find that when I move the mouse over to the right to grab the scrollbar I have to "move off" the edge of the screen by a few pixels, otherwise the mouse press yields a resizer (see literal screenshot - programmatic screenshots make the mouse disappear).

This is frustrating as normally you can just "throw" the mouse over to the right and be sure you can grab the scrollbar. But now I have to do a slight pull back first.

The app is installed with Snap and is fully maximised (tested that many times!).

![2021-09-11_13-39](https://user-images.githubusercontent.com/299102/132948145-8e351e90-e57d-4274-8689-818c479a9aa3.png)

**Comment by JaspalSuri** [comment]
Hi @hlascelles, sorry to hear that. Can you open a note, click on the **Options** menu, then check to see if **Margin Resizers** are (there is one resizer on each side of the note) enabled? If so, please click on it to disable it. If not, please let me know as I haven't been able to reproduce the issue yet.

**Comment by hlascelles** [comment]
Just had a go - they were off. I turned them on and checked it was pushed to the right as possible and turned them off again, and same problem.

EDIT: Well, this is a pickle. I turned on the chrome debug window to see if I could find out what is going on (I could not). I had to move it to the left side so it didn't interfere with the right side. However, now I cannot close it, or click on the menu icon.

See video:

https://user-images.githubusercontent.com/299102/133049152-dbff38cf-fbfd-4b01-a7aa-f79bbe365163.mp4

Do you have any testers who use Linux?

**Comment by JaspalSuri** [comment]
Thank you for the video! I'm glad to hear that restarting the application closed the developer tools view.

>Do you have any testers who use Linux?

I tested this on Ubuntu 20.04 LTS and wasn't able to reproduce the issue there. Can you try reproducing the issue in another desktop environment for Linux Mint?

**Comment by hlascelles** [comment]
I've been keeping an eye out but not found another Linux install I could test. It's still an issue though with v3.11.0.

I hear a major UI rewrite is coming. I will wait for that to drop before I dig deep into this. I can live with it for now.

**Comment by moughxyz** [comment]
> I hear a major UI rewrite is coming

Not true, for the record. We've been gradually improving the UI over the last year. The plan was always gradual incremental releases and not a major, single groundbreaking release.
